test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -4597,6 +4606,104 @@ public void testGetGroupsUserMembersOnly() throws Exception {
   }
 
   @Test
+  public void testGetGroups_fullFeeds() throws Exception {
+    insertUsers("User1", "User2", "User3", "User4", "User5");
+    insertGroup("Group1", "User1", "User2", "User3");
+    insertGroup("Group2", "User3", "User4", "User5");
+
+    ImmutableMap<GroupPrincipal, ? extends Collection<Principal>>
+        expected = ImmutableMap.of(
+            new GroupPrincipal("Group1", "NS_Local"),
+            ImmutableSet.<Principal>of(new UserPrincipal("User1", "NS"),
```

Comment:
These are hard to read. Brett aligned the arguments, which is a borderline style guide fail, IMHO. I would just put move User1 to the next line, as ditto for all the others.